fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,22 +57,56 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      requestOpenRollCall(event.idAlias).catch((e) =>
-        console.debug('Unable to send Roll call re-open request', e),
-      );
+      const time = Timestamp.EpochNow();
+      requestReopenRollCall(event.idAlias, time)
+        .then(() => {
+          if (!event.idAlias) {
+            console.debug(
+              'Unable to send roll call re-open request, the event does not have an idAlias',
+            );
+            return;
+          }
+          navigation.navigate(STRINGS.roll_call_open, {
+            rollCallID: event.id.toString(),
+            updateID: Hash.fromStringArray(
+              EventTags.ROLL_CALL,
+              lao.id.toString(),
+              event.idAlias.toString(),
+              time.toString(),
+            ),
+          });
+        })
+        .catch((e) => console.debug('Unable to send Roll call re-open request', e));
     } else {
```

Comment:
it seems that both branches of this if/else are doing very similar things. Try to see if they can be factored !